themes/tcars/theme.css

```diff
@@ -0,0 +1,420 @@
+
+                    & a  {
+                        min-height: 0;
+                        height: 50%;
+                    }
+                }
+
+                & li:nth-of-type(2) a {
+                    background-color: var(--trek-color-3);
+                    box-shadow: var(--quantum-glow);
+                }
+
+                & li:nth-of-type(1) a {
+                    background-color: var(--trek-color-2);
+                    min-height: 0;
+                    height: 50%;
+                    box-shadow: 0 0 12px var(--trek-color-2);
+                }
+
+                & li:nth-of-type(1)::before {
+                    background-color: var(--trek-color-2);
+                    bottom: 50%;
+                    box-shadow: 0 0 10px var(--trek-color-2);
+                }
+
+                & li:nth-of-type(1)::after {
+                    bottom: 50%;
+                }
+
+                & li:last-of-type > a {
+                    background-color: var(--trek-color-1);
+                }
+            }
+
+            &.curve-top-right {
+                & > a {
+                    border-top-left-radius: 100%;
+                }
+
+                & li:nth-of-type(1) a {
+                    background-color: var(--trek-color-1);
+                    min-height: 0;
+                    height: 50%;
+                }
+
+                & li:nth-of-type(1)::before {
+                    background-color: var(--trek-color-1);
+                    top: 50%;
+                    box-shadow: 0 0 10px var(--trek-color-1);
+                }
+
+                & li:nth-of-type(1)::after {
+                    top: 50%;
+                }
+
+                & li:nth-of-type(odd) a {
+                    min-height: 0;
+                    height: 50%;
+                }
+
+                & li:last-of-type > a {
+                    background-color: var(--trek-color-5);
+                    box-shadow: 0 0 12px var(--trek-color-5);
+                }
+            }
+
+            &:nth-of-type(1) a {
+                background-color: var(--trek-color-5);
+                box-shadow: 0 0 12px var(--trek-color-5);
+            }
+
+            &:nth-of-type(2) a {
+                background-color: var(--trek-color-2);
+                box-shadow: 0 0 12px var(--trek-color-2);
+            }
+
+            &:nth-of-type(4) a {
+                background-color: var(--trek-color-3);
+                box-shadow: var(--quantum-glow);
+            }
+
+            &:nth-of-type(5) a {
+                background-color: var(--trek-color-4);
+                box-shadow: 0 0 12px var(--trek-color-4);
+            }
+
+            &:nth-of-type(6) a {
+                background-color: var(--trek-color-2);
+                box-shadow: 0 0 12px var(--trek-color-2);
+            }
+        }
+    }
+}
+
+/* Holographic text effects for headers */
+h1, h2, h3 {
+    animation: holographic-flicker 6s ease-in-out infinite;
```

Comment:
I don't want glow on the heading elements, maybe this flicker could be on other elements with a box-shadow? or maybe a different animation.